sentry/src/main/java/io/sentry/protocol/SentryStackTrace.java

```diff
@@ -66,6 +66,16 @@ public final class SentryStackTrace implements JsonUnknown, JsonSerializable {
    */
   private @Nullable Boolean snapshot;
 
+  /**
+   * This value indicates if, and how, `instruction_addr` values in the stack frames need to be
+   * adjusted before they are symbolicated. TODO: should we make this an enum or is a string value
+   * fine?
```

Comment:
Since this is part of the public API, it is also a sensible decision to make now.